vmm/src/device_manager.rs

```diff
@@ -464,6 +462,7 @@ pub enum DeviceManagerError {
     /// Cannot hotplug device behind vIOMMU
     InvalidIommuHotplug,
 
+    #[cfg(target_arch = "x86_64")]
```

Comment:
I don't think this error is used anymore? Can you just delete it?

Reply:
Yes, should be removed.